setup.py

```diff
@@ -67,15 +67,15 @@ def makeExtension(extName, files=None):
 elif platform == 'ios':
     include_dirs = [
             join(kivy_ios_root, 'build', 'include'),
-            join(kivy_ios_root, 'build', 'include', 'SDL')]
+            join(kivy_ios_root, 'build', 'include', 'SDL2')]
```

Comment:
I'm not sure about this one, but it might be even a good fix. Kivy-ios recipe has `SDL2`, however there it's just `SDL`. I wonder how does it even compile for iOS with mixed libs. @akshayaurora @Zen-CODE ?

Reply:
Yes, agreed. These changes seem specifically targeted at Android, so changes for other OS's should be avoided.@snuq. Have you tested these changes on Mac/iOS?

Reply:
I did not test it, no, i guess i got carried away changing all the instances of SDL